mouse-click-effects@anaximeno/files/mouse-click-effects@anaximeno/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "uuid": "mouse-click-effects@anaximeno",
+    "name": "Mouse Click Effects",
+    "version": "0.1.0",
+    "description": "Display mouse click effects on Cinnamon.",
+    "url": "https://github.com/anaximeno/mouse-click-effects",
+    "website": "https://github.com/anaximeno/mouse-click-effects",
+    "cinnamon-version": [
+        "5.4"
```

Comment:
```suggestion
        "5.4",
        "5.6",
        "5.8",
        "6.0",
        "6.2"
```

Reply:
I think it needs this in order to be installed beyond Cinnamon 5.4. (We should probably remove this check and just use the folder structure like we do for Applets.)

Reply:
Yes, initially, I thought it would work similarly to the applets, but it seems that it doesn't work that way yet, it might be a possible future update. Thanks!

Reply:
The code for the check is in Cinnamon's code. I think it was put there before the folder structuring was implemented. If so, we can probably remove the check. Things don't break as often in Cinnamon to warrant a new version for every extension for every release IMHO.

Reply:
It is my understanding that for applets it can work with both of the platform versioning schemes, for example, to make it work with the folder structure you just would have to pass an entry `"multiversion": true` in metadata.json, so we probably could do something similar to extensions, this way we can also still keep extensions that won't be adapted for next versions still available for prior versions (maybe using the current versioning scheme).

Reply:
Hmm. I wasn't aware that `multiversion` was still used too. It would be good to know and create a consistent ecosystem around it. I feel like a lot has changed as time has gone on and there might be some legacy components still lurking around.